scalding-serialization-macros/src/test/scala/com/twitter/scalding/serialization/macros/MacroOrderingProperties.scala

```diff
@@ -238,40 +248,49 @@ class MacroOrderingProperties extends FunSuite with PropertyChecks with ShouldMa
   test("Test out jl.Byte") {
     implicit val a = arbMap { b: Byte => java.lang.Byte.valueOf(b) }
     check[java.lang.Byte]
+    checkCollisions[java.lang.Byte]
```

Comment:
actually, if we are hashing 256 things into 32 bits, it should probably not collide actually.

So, this test is fine (assuming it passes).